suitcase/dataexchange/__init__.py

```diff
@@ -262,7 +262,7 @@ def stop(self, doc):
         del self._image_timestamps[-2 * self._chunk_size:]
 
         theta = np.interp(
-            self._image_timestamps,
+            [item + 631152000 for item in self._image_timestamps],
```

Comment:
"Magic numbers" should be labeled.
```suggestion
            EPICS_TO_UNIX_OFFSET = 631152000  # 20 years in seconds
            [item + EPICS_TO_UNIX_OFFSET for item in self._image_timestamps],
```